akka-docs/src/main/paradox/stream/operators/RestartSource/onFailuresWithBackoff.md

```diff
@@ -14,11 +14,41 @@ Wrap the given @apidoc[Source] with a @apidoc[Source] that will restart it when
 
 ## Description
 
- This @apidoc[Source] will never emit a failure, since the failure of the wrapped @apidoc[Source] is always handled by
- restarting. The wrapped @apidoc[Source] can be cancelled by cancelling this @apidoc[Source].
- When that happens, the wrapped @apidoc[Source], if currently running will be cancelled, and it will not be restarted.
- This can be triggered simply by the downstream cancelling, or externally by introducing a @apidoc[KillSwitch] right
- after this @apidoc[Source] in the graph.
+This @apidoc[Source] will never emit a failure, since the failure of the wrapped @apidoc[Source] is always handled by
+restarting. The wrapped @apidoc[Source] can be cancelled by cancelling this @apidoc[Source].
+When that happens, the wrapped @apidoc[Source], if currently running will be cancelled, and it will not be restarted.
+This can be triggered simply by the downstream cancelling, or externally by introducing a @apidoc[KillSwitch] right
+after this @apidoc[Source] in the graph.
+
```

Comment:
Maybe a "see also" ref-links for 1..* of:  `RestartSource.withBackoff`, `RestartFlow`, `RestartWithBackoffFlow`, `RestartSink`. Or are they in the TOC anyway?